hudi-utilities/src/main/java/org/apache/hudi/utilities/config/CloudSourceConfig.java

```diff
@@ -87,6 +87,13 @@ public class CloudSourceConfig extends HoodieConfig {
       .markAdvanced()
       .withDocumentation("Only selects objects in the bucket whose relative path starts with this prefix");
 
+  public static final ConfigProperty<String> SELECT_RELATIVE_PATH_REGEX = ConfigProperty
+      .key(STREAMER_CONFIG_PREFIX + "source.cloud.data.select.path.regex")
+      .noDefaultValue()
+      .withAlternatives(DELTA_STREAMER_CONFIG_PREFIX + "source.cloud.data.select.path.regex")
```

Comment:
Rename, since it's relative path?
```suggestion
      .withAlternatives(DELTA_STREAMER_CONFIG_PREFIX + "source.cloud.data.select.relative.path.regex")
```